_unittest_ironpython/test_15_ibs_reader.py

```diff
@@ -0,0 +1,24 @@
+from conf_unittest import test_generator, PytestMockup
+import os
+
+test_filter = "test_"
+
+test_name = os.path.basename(__file__).replace(".py", "")
+mymodule = __import__("_unittest." + test_name, fromlist=["TestClass"])
+test_obj = mymodule.TestClass()
+
+
+class TestSequenceFunctionsGenerate(PytestMockup):
+    @classmethod
+    def setUpClass(cls):
+        test_obj.setup_class()
+
+    @classmethod
+    def tearDownClass(cls):
+        test_obj.teardown_class()
```

Comment:
Method names, even within our unit tests, should conform to PEP8. These should be pascal case.

Reply:
@akaszynski The camelCase naming of the class methods comes from the `PytestMockup` class inherited from `TestCase` in the `unittest` module.
Those methods are overridden.